test/led.js

```diff
@@ -361,14 +361,16 @@ exports["Led - PWM (Analog)"] = {
 
     this.led.mode = 1;
     this.led.pulse();
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
 
     this.led.mode = 1;
     this.led.fade();
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
 
     this.led.strobe();
-    // pre 0.8.13, this would change to 1
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
```

Comment:
I hear you, I was thinking "this isn't the right way to do this" as I was
doing it. I'll give it some more thought and see if I can come up with a
better solution.

On Wed, Jan 21, 2015 at 3:12 PM, Rick Waldron <notifications@github.com>
wrote:

> In test/led.js
> <https://github.com/rwaldron/johnny-five/pull/586#discussion_r23330317>:
>
> >      test.equal(this.led.mode, 3);
> >
> >      this.led.strobe();
> > -    // pre 0.8.13, this would change to 1
> > +    this.clock.tick(1001);
> >      test.equal(this.led.mode, 3);
>
> I get it now. I'm not sure I like it, but I'm not going to block on that
> either.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rwaldron/johnny-five/pull/586/files#r23330317>.
>